specs/core/1_shard-data-chains.md

```diff
@@ -122,3 +131,112 @@ Verify that the `shard_block_combined_data_root` is the output of these function
 ### Shard block fork choice rule
 
 The fork choice rule for any shard is LMD GHOST using the validators currently assigned to that shard, but instead of being rooted in the genesis it is rooted in the block referenced in the most recent accepted crosslink (ie. `state.crosslinks[shard].shard_block_hash`). Only blocks whose `beacon_chain_ref` is the block in the main beacon chain at the specified `slot` should be considered (if the beacon chain skips a slot, then the block at that slot is considered to be the block in the beacon chain at the highest slot lower than a slot).
+
+## Changes to beacon chain
+
+Change to attestation verification:
+
+* Let `expected_depth = log2(SHARD_BLOCK_SIZE // SHARD_CHUNK_SIZE * next_power_of_2(slot - last_crosslink_slot))`. For each integer `i = 0, 1`, let `messages[i]` be `AttestationSignedData(slot, shard, parent_hashes, shard_block_hash, i == 1, expected_depth, attestation_indices.total_validator_count, justified_slot)`.
```

Comment:
1. `AttestationSignedData` was renamed to `AttestationData`, and some fields were updated.
2.  What's the `i == 1` here?